### PR TITLE
roles: pgbouncer: drop `-q` option from call

### PR DESCRIPTION
**Pull request by jimnydev**
Drop unnecessary  `q`  option

### CODE REVIEW

File: roles/pgbouncer/config/tasks/main.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@
       become: true
       become_user: postgres
       ansible.builtin.command: >-
-        {{ postgresql_bin_dir }}/psql -p {{ postgresql_port }} -U {{ patroni_superuser_username }} -d postgres -tAXcq
+        {{ postgresql_bin_dir }}/psql -p {{ postgresql_port }} -U {{ patroni_superuser_username }} -d postgres -tAXqc
```
Comment (vitabaks) [inline roles/pgbouncer/config/tasks/main.yml:29]:
```suggestion
        {{ postgresql_bin_dir }}/psql -p {{ postgresql_port }} -U {{ patroni_superuser_username }} -d postgres -tAXc
```

I think the `-q `option is not required when using -`tAXc`

Comment (kornrunner) [reply roles/pgbouncer/config/tasks/main.yml]: I haven't checked in detail, just tried to make the change least disruptive. 

Nevertheless, I dropped the `-q` option in following commit. Thanks!

Comment (jimnydev) [reply roles/pgbouncer/config/tasks/main.yml]: I haven't checked in detail, just tried to make the change least disruptive.

Anyhow, I've dropped the `-q` option in following update. Thanks!

Comment (vitabaks) [reply roles/pgbouncer/config/tasks/main.yml]: @jimnydev Thanks. Please test it.

Comment (jimnydev) [reply roles/pgbouncer/config/tasks/main.yml]: It took me a bit, but I did test both changes (swapping the options and dropping `q) - both are working fine. Thank you for merging it!

**Pull request merged**